[docs] Add material-ui-pickers in pickers page (#13697)
mui-org/material-ui: docs/src/pages/demos/pickers/pickers.md

A native time picker example with `type="time"`:

{{"demo": "pages/demos/pickers/TimePickers.js"}}

## MUI-pickers example

> Material-ui-pickers package provides more complex ui for date & time controls. You can refer [documentation website](https://material-ui-pickers.firebaseapp.com/) for more information.

Reviewer: ```suggestion
[material-ui-pickers](https://material-ui-pickers.firebaseapp.com/) provides date and time controls that follow the Material Design spec.
```
